test_ppod.py

```diff
@@ -99,3 +101,29 @@ def test_filter_files_in_bucket_without_matching_file(mocked_s3):
     with pytest.raises(KeyError):
         files = filter_files_in_bucket("ppod", "download/")
         next(files)
+
+
+def test_post_files_to_pod_success(mocked_pod):
+    with open("fixtures/pod_with_namespaces.xml", "rb") as pod_with_namespaces, open(
+        "fixtures/pod_response.html", "r"
+    ) as pod_response:
+        response = post_files_to_pod(
+            "http://example.example/organizations/ORG/uploads?stream=default",
+            {"Authorization": "Bearer 1234abcd"},
+            "pod_file",
+            pod_with_namespaces,
+        )
+        assert response.status_code == 200
+        assert response.text == pod_response.read()
+
+
+def test_post_files_to_pod_bad_url_raises_error():
+    with pytest.raises(requests.exceptions.ConnectionError), open(
+        "fixtures/pod_with_namespaces.xml", "rb"
+    ) as pod_with_namespaces:
+        post_files_to_pod(
+            "http://example.example/organizations/ORG/uploads?stream=default",
```

Comment:
When this test is run it attempts to make a real call to this url, which is not good practice as we want everything in tests to be mocked and under our control. Also, it was confusing at first because this url is the same url as the test above. What you want to do instead is also mock a bad url and have that return an error response.